lib/build/raty.js

```diff
@@ -711,8 +711,7 @@ var Raty = function () {
       if (name) {
         var icon = this.opt[name];
         var star = this.stars[Math.ceil(score) - 1];
-
-        this._setIcon(star, icon);
+        star && this._setIcon(star, icon);
```

Comment:
Do you have an example where the star is `undefined`? If yes, could you tell me, because it should be covered by a test to simulate it.

Reply:
it gives star undefined when (Math.ceil(score)-1) < 0

Reply:
Yeah, but do you know in which situation (use case)? Would be nice to cover it in the tests. Since it's a private method, something is happening in another place to call it wrong.

Reply:
when score is < 1 and >0